### DISCUSSION TITLE
AdNauseam Settings (i.e. 3 activation sliders) are Popping Up everytime I open Firefox (up to date).

### DISCUSSION THREAD

**Discussion by Acleacius** [dhowe/AdNauseam#2563] [Q&A]
I think it's been happening since the most recent update 8/24 and sadly it gets so annoying I disable AdNauseam.
Firefox is set at Default Privacy but personally I'm overly cautious about Cookies and Trackers.
So I close Firefox often and use CCleaner about 5 times a day. Iirc, AdNauseam didn't do this in the past, so not sure if it's a bug.

Anyone tell me how to fix, workaround or hack AdNauseam so it stops asking me to set it, each time I open Firefox?

Thanks

**Comment by Vanessalf-nita** [comment]
I'm having the same problem.
